packages/middleware/src/apiClientFactory/applyContextToApi.ts

```diff
@@ -1,4 +1,5 @@
 // @ts-check
+import { isCustomEndpointHandler } from "../helpers";
```

Comment:
```suggestion
import { isExtensionEndpointHandler } from "../helpers";
```
Because we also use extensions which are not a custom code I think - custom = client?